packages/react-heartwood-components/.storybook/data/eventDetails.js

```diff
@@ -0,0 +1,571 @@
+const avatar =
+	'https://images.unsplash.com/photo-1542080681-b52d382432af?ixlib=rb-1.2.1&auto=format&fit=crop&w=96&h=96&q=80'
+
```

Comment:
@corbanb @kengoldfarb I'm hoping these can shed some light on what needs to change in the API. We'll also need calendar events to return a title, but the component to render it doesn't exist yet. Most of the heavy lifting is done by `items` and their props (`componentProps`) align with the components they render.